rosetta/tests/mirror-only-distribution.sh

```diff
@@ -6,23 +6,38 @@ cd $SCRIPT_DIR
 set -eou pipefail
 
 # Version should work on Linux || Darwin
-#repo_tmp=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
-repo_tmp=/tmp/t5x
-patchlist_tmp=$(mktemp /tmp/patchlist.txt.XXXXXX)
+tmp_base=$(mktemp -d 2>/dev/null || mktemp -d -t 'upstream')
```

Comment:
I do see code duplication: `mirror-only-discribution.sh` is identical to `upstream-only-distribution.sh` (except maybe couple lines of code). Do you plan to merge them?